modules/nf-core/kallistobustools/ref/main.nf

```diff
@@ -62,4 +62,33 @@ process KALLISTOBUSTOOLS_REF {
         END_VERSIONS
         """
     }
+
+    stub:
+    def args = task.ext.args ?: ''
```

Comment:
```suggestion
```

Unused variable like above

Reply:
probably just putting $args into the kb ref call.

Reply:
Same reason as the other comment, the `$args` is not used in the below `stub:` code, only in the `script:` block (which has its own variable definitions).